.github/workflows/sqflite.yml

```diff
@@ -12,8 +12,8 @@ on:
       - '.github/workflows/analyze.yml'
       - '.github/actions/flutter-test/**'
```

Comment:
Could we also change to dart-test here, if `sqflite` is not dependent on flutter?